### ISSUE TITLE
.usa-logo-text only supports short titles

### ISSUE THREAD

**Issue by phae** [uswds/uswds#1729]
It seems that the pattern for headers doesn't work for text over a certain length.  As soon as the words are too long, the title pops off the top of the layout (or up against the banner if that's turned on, as seen in your docs). 

See screen shot.

<img width="554" alt="screenshot 2017-02-22 14 34 35" src="https://cloud.githubusercontent.com/assets/44568/23235911/1c715c14-f90c-11e6-89f8-4592c1b9200f.png">
<img width="839" alt="screenshot 2017-02-22 14 34 08" src="https://cloud.githubusercontent.com/assets/44568/23235913/1d8ced8e-f90c-11e6-93e5-9104886f6f4b.png">

SSs are Chrome 56.0.2924.87 OSX 10.10.5

**Comment by juliaelman** [comment]
Thank you for your interest in the Draft U.S. Web Design Standards! We have received notification of your issue and will add it to our backlog for future feature consideration.

In the meantime, you can solve this issue by overriding the `.usa-navbar` styles. It looks like you are using the Basic Header, is that accurate?

Also, I am not sure if you are using Sass or CSS, but here are some solves I tried locally that seem to resolve the issue (see screenshot below FYR):

**Sass**
```
// path/to/my/scss/files/main/scss/my-custom_vars.scss

.usa-header-basic
.usa-navbar {
  @include media($nav-width) {
    width: 25%;
  }
}
```

**CSS**
```
@media screen and (min-width: 951px)
.usa-header-basic .usa-navbar {
  width: 25%;
}
```

![screen shot 2017-02-23 at 10 06 21 am](https://cloud.githubusercontent.com/assets/955558/23264569/c6d5aac2-f9af-11e6-808a-873ca7ca6d16.png)

**Comment by phae** [comment]
Yes, the basic header.

In terms of what we're using in addition, so far literally neither CSS nor SaSS. We've only used the pattern library for a demo. 

Also, I don't think this is a feature request. It's a bug. Not sure what the longest Federal Agency name is, but might be worth putting that in any tests you have.

**Comment by juliaelman** [comment]
@phae thanks for the clarification! Agreed that this is a bug not a feature and appreciate you pointing this out to us to help resolve.

**Comment by maya** [comment]
Fixed in #2750.
